pkgs/applications/editors/emacs/elisp-packages/session-management-for-emacs/default.nix

```diff
@@ -17,8 +18,8 @@ stdenv.mkDerivation {
 
   meta = {
     /* installation: add to your ~/.emacs
-       (require 'session)
-       (add-hook 'after-init-hook 'session-initialize)
+      (require 'session)
```

Comment:
line comments would probably be better here

Reply:
like this?

Reply:
yes